### PR TITLE
[ACR-1151, ACR-1171] Update DALi documents about video view property for Tizen 4.0 

### PR DESCRIPTION
**Pull request by JoogabYun**
### Change Description ###

Update DALi documents about video view property for Tizen 4.0 

### API Changes ###

 - ACR: ACR-1151, ACR-1171

### CODE REVIEW

File: docs/application/native/guides/ui/dali/videoview.md [hunk 1]
```diff
@@ -108,6 +110,16 @@ newMap.Insert( "volumeRight", 0.5f );
 mView.SetProperty( VideoView::Property::VOLUME, newMap );
 ```
 
+How to use DISPLAY_MODE.
+
+there are enum of display modes: LETTER_BOX, ORIGIN_SIZE, FULL_SCREEN, CROPPED_FULL, ORIGIN_OR_LETTER, DST_ROI (See [multimedia display](https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.mobile.apireference/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html))
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/dali/videoview.md]: Hello, @JoogabYun 

How about changing the update by you in below form?
The re-written below needs your check. If you think this modification is acceptable, please update the PR.

---

To change the display mode, use the `PLAYER_DISPLAY_MODE` enums (in [mobile](../../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html#gaafcf6f7487d8cb376e1851a6e1253f3a) and [wearable](../../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html#gaafcf6f7487d8cb376e1851a6e1253f3a) applications) .
 - `LETTER_BOX`
 - `ORIGIN_SIZE`
 - `FULL_SCREEN`
 - `CROPPED_FULL`
 - `ORIGIN_OR_LETTER`
 - `DST_ROI`
```
mView.SetProperty( VideoView::Property::DISPLAY_MODE, VideoView::DisplayMode::LETTER_BOX );
```

---

Hello, @annie-samsung 
If, JoogabYun agrees with this modification, in that case please review the sentence of above.
If he doesn't accept it, then please review the original update by JoogabYun.

File: docs/application/native/guides/ui/dali/videoview.md [hunk 2]
```diff
@@ -108,6 +110,19 @@ newMap.Insert( "volumeRight", 0.5f );
 mView.SetProperty( VideoView::Property::VOLUME, newMap );
 ```
 
+To change the display mode, use the `PLAYER_DISPLAY_MODE` enums (in [mobile](https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.mobile.apireference/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html) and [wearable](https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.wearable.apireference/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html) applications) .
```
Comment (journey2w) [inline docs/application/native/guides/ui/dali/videoview.md]: We don't provide the links to https://developer.tizen.org anymore.

Please see API Reference links in this page and change this sentence like other links.
For instance, we are using like below:
```
See the Player API Reference (in [mobile](../../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) applications).    
```

File: docs/application/native/guides/ui/dali/videoview.md [hunk 3]
```diff
@@ -108,6 +110,19 @@ newMap.Insert( "volumeRight", 0.5f );
 mView.SetProperty( VideoView::Property::VOLUME, newMap );
 ```
 
+To change the display mode, use the `PLAYER_DISPLAY_MODE` enums (in [mobile](../../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) applications) .
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/dali/videoview.md]: One additional point to check : The link was to "group__CAPI__MEDIA__PLAYER__**DISPLAY**__MODULE.html" at first, and now it is heading to "group__CAPI__MEDIA__PLAYER__MODULE.html". Is `PLAYER_DISPLAY_MODE` really linked to "group__CAPI__MEDIA__PLAYER__MODULE.html", not  "group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html"?

**Comment by mijin85cho** [comment]
Hello, @JoogabYun,
Would you please check the comments?

**Comment by mijin85cho** [comment]
Thanks for the update.

@journey2w and @annie-samsung, I hope you can re-check the latest version.

**Comment by mijin85cho** [comment]
Thanks for the review & update.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**